experiments/storm/src/test/scala/io/gearpump/experiments/storm/GearpumpThriftServerSpec.scala

```diff
@@ -18,7 +18,7 @@
 
 package io.gearpump.experiments.storm
 
-import org.apache.thrift7.server.TServer
+import backtype.storm.security.auth.ThriftServer
```

Comment:
Since you changed the thrift server package, have you tested whether this impact the storm 0.9 compatibility?